src/common/event-utils/types.ts

```diff
@@ -9,3 +12,16 @@ type NavigationKeysEdge = 'Down' | 'Left' | 'Right' | 'Up' | 'Esc'
 // 'Separator' | '0' | '1' | '2' | '3' | '4' | '5' | '6' | '7' | '8' | '9'
 
 export type Key = ModifierKeys | NavigationKeys | NavigationKeysEdge
+
+type BaseEventHandler<E extends SyntheticEvent<any>, P> = (event: E, props?: P) => void;
+
+export type EbayEventHandler<ElementType = Element, PropsObject = Record<string, any>> =
```

Comment:
i am not using this for now -- just picked from the other branch.

Reply:
removed

Reply:
ah! i was removing it from menu comp types file. :)

Reply:
https://github.com/eBay/ebayui-core-react/pull/231/commits/1b64e1aed8d2ffa244686fbc16c0d988ec03fa8e